pyproject.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT"
 readme = "docs/README.md"
 repository = "https://github.com/Safe-DS/Stdlib"
 documentation = "https://safe-ds-stdlib.readthedocs.io"
-keywords = ["data-science", "machine-learning", "usability", "learnability"]
```

Comment:
Should not be changed.

Reply:
Please read the comments and either write why you don't want to work on it or fix them. Don't just resolve them without doing anything.